src/main/java/de/tum/in/www1/bamboo/server/ServerNotificationTransport.java

```diff
@@ -488,18 +486,14 @@ private JSONObject createLogEntryJSONObject(LogEntry logEntry) throws JSONExcept
     private void logToBuildLog(String s) {
         if (buildLoggerManager != null && plan != null) {
             BuildLogger buildLogger = buildLoggerManager.getLogger(plan.getPlanKey());
-            if (buildLogger != null) {
-                buildLogger.addBuildLogEntry("[BAMBOO-SERVER-NOTIFICATION] " + s);
-            }
+            buildLogger.addBuildLogEntry("[BAMBOO-SERVER-NOTIFICATION] " + s);
```

Comment:
Have you removed the null-check on purpose?

Reply:
Yes, `buildLoggerManager.getLogger(plan.getPlanKey())` never returns null as it is annotated with @nonNull